gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/BundleMapperService.java

```diff
@@ -116,6 +119,9 @@ public Bundle mapToBundle(RCMRIN030000UK06Message xmlMessage) {
             mapObservationComments(ehrExtract, patient, encounters, practiseCode);
         addEntries(bundle, observationComments);
 
+        var documentReferences = mapDocumentReferences(ehrExtract, patient, encounters, practiseCode);
```

Comment:
You will need to add a check that this mapper has been called in the BundleMapperServiceTest. Could you also add an 2 examples of NarrativeStatement -> DocumentReferences in the integration test. One at the top level of an ehrComposition and another in a nested compound statement?